src/tokens.py

```diff
@@ -1,4 +1,30 @@
+import sys
+sys.path.append(
+    "calculator_lexical_analyzer/src"
+)
+from validator import Validator
+
+
 class Tokens:
-    def __init__(self):
-        self.token_list = []
-        
\ No newline at end of file
+
+    def split_tokens(self, input: str) -> list:
+        token_list = []
+        validator = Validator()
+        token = ""
+        for c in self.input:
```

Comment:
Isso aqui não devia ser input?

Reply:
De fato é isso mesmo, antes esse input estava no próprio objeto, aí refatorando eu esqueci de alterar.
```suggestion
        for c in input:
```